lib/tasks/one_time/2017_Q4/payments.rake

```diff
@@ -0,0 +1,89 @@
+# PREREQUISITE MIGRATIONS:
+# 20171013141538_create_payments
+# 20171026103648_add_dates_to_payments
```

Comment:
If these are prerequisites, then you need to check for them.  (And if the prerequisites are not satisfied, return some sort of informative error message, obv.)

Reply:
Yes, that is a good idea.  I'd like to hold off on this for this PR, since

1) These tasks will fail - without altering the data - if any pre-requisite migrations have not been run, and,

2) Checking the pre-reqs is somewhat involved, including one for more of: a) checking presence of the migration file in the env, b) confirming the presence / absence of any schema element added / dropped, and c) confirming the migration has actually been run.

If we decide to go with these rake tasks as a way to manage production data changes when we can make these changes in a separate PR. (and, yes, I realize that doing data changes _in_ a data migration makes some of this complexity go away ... to be discussed further).